diff --git a/dist/calendar.d.ts b/dist/calendar.d.ts
index 62fde6d8974bbf10a5fe76b22a65f22210822c96..b3e91b345520fb0ad1a5108a3b212f7eebbf91c4 100644
--- a/dist/calendar.d.ts
+++ b/dist/calendar.d.ts
@@ -7,7 +7,7 @@ import MonthYearDropdown from "./month_year_dropdown";
 import Time from "./time";
 import Year from "./year";
 import YearDropdown from "./year_dropdown";
-import type { Day } from "date-fns/types";
+import type { Day } from "date-fns";
 interface YearDropdownProps extends React.ComponentPropsWithoutRef<typeof YearDropdown> {
 }
 interface MonthDropdownProps extends React.ComponentPropsWithoutRef<typeof MonthDropdown> {
